src/include/users/widgets.rb

```diff
@@ -1851,20 +1737,13 @@ def StoreDefaults(key, event)
       event = deep_copy(event)
       defaults = Users.GetLoginDefaults
       new_defaults = {}
-      Builtins.foreach(["home", "shell", "skel", "inactive", "umask"]) do |key2|
+      Builtins.foreach(["home", "shell", "inactive", "umask"]) do |key2|
```

Comment:
when I see that umask, is not that now readonly?

Reply:
It is not managed by *useradd* anymore. Now, that value is in *login.defs* and it can be read and written.

Reply:
See https://github.com/yast/yast-users/blob/y2users/src/lib/y2users/useradd_config.rb#L108-L123.